pkg/crc/machine/start.go

```diff
@@ -719,3 +723,13 @@ func updateKubeconfig(ctx context.Context, ocConfig oc.Config, sshRunner *crcssh
 	}
 	return nil
 }
+
+func bundleMismatchWithPreset(preset string, bundleMetadata *bundle.CrcBundleInfo) error {
+	if preset != string(crcPreset.OpenShift) && bundleMetadata.IsOpenShift() {
+		return errors.Errorf("Preset %s is used but bundle is provided for %s preset", crcPreset.Podman, crcPreset.OpenShift)
+	}
+	if preset != string(crcPreset.Podman) && !bundleMetadata.IsOpenShift() {
```

Comment:
@cfergeau we can but we shouldn't because our openshift bundle is based on RHCOS which have older version of podman and podman bundle is based on FCOS so better to differ those use cases.

Reply:
Actually, it might still be worth to explore, last time we discussed this, it was not clear if there were any known issues for our use cases, so even if the podman version is slightly old, it might still do the trick.

Reply:
Continue conversation on: https://github.com/code-ready/crc/issues/2867

For now we tie this choice to a bundle, as also on the welcome screen it will be a choice to use Podman or OpenShift